ssl/ssl_cert.c

```diff
@@ -1152,16 +1175,22 @@ int ssl_cert_lookup_by_nid(int nid, size_t *pidx)
             return 1;
         }
     }
-
+    for (i = 0; i < ctx->sigalg_list_len; i++) {
+        if (ctx->ssl_cert_info[i].nid == nid) {
+            *pidx = SSL_PKEY_NUM+i;
```

Comment:
Nit: Add spaces around `+`

Reply:
So changed in 0ac814f8247d1a5f157eec2b62bce0000c3a2d1e.